uber/templates/base.html

```diff
@@ -219,7 +159,39 @@
                             </ul>
                         </li>
                     </ul>
-                    <ul class="nav navbar-nav" id="main-menu"></ul>
+                    {% block mainmenu %}
+                    <ul class="nav navbar-nav" id="main-menu">
+                        {% block mainmenu_items %}
```

Comment:
actually a non-obvious thing about this extra block here is the later, in the jinja2 refactor, we can just override the ```mainmenu``` block with nothing, thereby disabling the main menu. (or, more likely, include this block only in admin pages).

right now we do this with the 'backlink' block, though hoping maybe we can make all this cleaner later on